Description: Replace references to pcsc headers with direct values
 Protocol should be handled in the connection plugin, not in the application
 or globalplatform library. Drop references to SCARD_PROTOCOL_T* names from
 generic globalplatform library.
Author: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>

---

--- globalplatform-6.0.0.orig/src/globalplatform/connection.h
+++ globalplatform-6.0.0/src/globalplatform/connection.h
@@ -41,8 +41,8 @@ extern "C"
 #define OPGP_TRACE_MODE_ENABLE 1 //!< Switch trace mode on
 #define OPGP_TRACE_MODE_DISABLE 0 //!< Switch trace mode off
 
-#define OPGP_CARD_PROTOCOL_T0 SCARD_PROTOCOL_T0 //!< Transport protocol T=0
-#define OPGP_CARD_PROTOCOL_T1 SCARD_PROTOCOL_T1 //!< Transport protocol T=1
+#define OPGP_CARD_PROTOCOL_T0 1 //!< Transport protocol T=0
+#define OPGP_CARD_PROTOCOL_T1 2 //!< Transport protocol T=1
 
 /**
  * Structure for holding all connection related functions for connection plugin libraries.
